src/components/MissingSidecar/MissingSidecar.tsx

```diff
@@ -17,7 +19,8 @@ type MissingSidecarProps = {
 class MissingSidecar extends React.Component<MissingSidecarProps, {}> {
   static defaultProps = {
     text: 'Missing Sidecar',
-    textTooltip: 'Missing Sidecar',
+    textTooltip:
+      "Istio container doesn't found in Pod(s). Check if the istio-injection label/annotation is correctly set on the namespace/workload.",
```

Comment:
grammar --> "Istio container not found in Pod(s)."    Or perhaps "Istio sidecar container not found in Pod(s)."